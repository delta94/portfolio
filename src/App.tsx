import 'bootstrap/dist/css/bootstrap.min.css';
import React, { Context, createContext, useEffect, useState } from 'react';
import { createUseStyles } from 'react-jss';
import Loader from 'react-loader-spinner';
import 'react-loader-spinner/dist/loader/css/react-spinner-loader.css';
import { getInformations } from './api/api';
import { AppBar } from './components';
import { About, Contact, Education, Experience, Home, Photography } from './screens';
import { CustomTheme, theme } from './theme';
import { Informations } from './utils/informations';

const useStyles = createUseStyles((theme: CustomTheme) => ({
  '@global': {
    body: {
      padding: 0,
      margin: 0,
      backgroundColor: theme.colors.primaryColor,
      fontFamily: theme.fontProperties.fontFamily,
      scrollBehavior: 'smooth',
    },
  },
  loaderContainer: {
    display: 'flex',
    width: '100vw',
    height: '100vh',
    justifyContent: 'center',
    alignItems: 'center',
  },
}));

const InformationsContext: Context<Informations | undefined> = createContext<Informations | undefined>(undefined);

const App: React.FC = () => {
  useStyles();
  const classes: Record<'@global' | 'loaderContainer', string> = useStyles();
  const [informations, setInformations] = useState<Informations | undefined>();
  const [showLoader, setShowLoader] = useState<boolean>(true);
  useEffect(() => {
    getInformations()
      .then((response) => {
        response.status === 200 && setInformations(response.data);
      })
      .catch((err) => console.warn(err))
      .finally(() => setShowLoader(false));
  }, []);

  const Components: JSX.Element[] = [<Home />, <About />, <Education />, <Experience />, <Photography />, <Contact />];
  return (
    <>
      <InformationsContext.Provider value={informations}>
        {showLoader ? (
          <div className={classes.loaderContainer}>
            <Loader type='Puff' color={theme.colors.secondaryColor} height={200} width={200} visible={showLoader} />
          </div>
        ) : (
          <>
            <AppBar />
            {Components.map((Component: JSX.Element, index: number) => (
              <div key={`${index}`}>{Component}</div>
            ))}
          </>
        )}
      </InformationsContext.Provider>
    </>
  );
};

export default App;
export { InformationsContext };
