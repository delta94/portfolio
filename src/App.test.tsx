import { render } from '@testing-library/react';
import React from 'react';
import ReactGa from 'react-ga';
import App from './App';
import { TestContextProvider } from './TestContextProvider';

test('App renders correctly', () => {
  ReactGa.initialize('test', { testMode: true });
  const { asFragment } = render(
    <TestContextProvider>
      <App />
    </TestContextProvider>,
  );
  expect(asFragment()).toMatchSnapshot();
});
