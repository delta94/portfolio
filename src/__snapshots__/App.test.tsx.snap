// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders correctly 1`] = `
<DocumentFragment>
  <div
    class="sticky-0-2-6"
  >
    <div
      class="mainContainer-0-2-1"
    >
      <div>
        <a>
          <div
            class="mainContainer-0-2-7"
          >
            <svg
              fill="currentColor"
              height="1em"
              stroke="currentColor"
              stroke-width="0"
              viewBox="0 0 576 512"
              width="1em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M280.37 148.26L96 300.11V464a16 16 0 0 0 16 16l112.06-.29a16 16 0 0 0 15.92-16V368a16 16 0 0 1 16-16h64a16 16 0 0 1 16 16v95.64a16 16 0 0 0 16 16.05L464 480a16 16 0 0 0 16-16V300L295.67 148.26a12.19 12.19 0 0 0-15.3 0zM571.6 251.47L488 182.56V44.05a12 12 0 0 0-12-12h-56a12 12 0 0 0-12 12v72.61L318.47 43a48 48 0 0 0-61 0L4.34 251.47a12 12 0 0 0-1.6 16.9l25.5 31A12 12 0 0 0 45.15 301l235.22-193.74a12.19 12.19 0 0 1 15.3 0L530.9 301a12 12 0 0 0 16.9-1.6l25.5-31a12 12 0 0 0-1.7-16.93z"
              />
            </svg>
            <div
              class="title-0-2-8 title-d0-0-2-9"
            >
              Home
            </div>
          </div>
        </a>
      </div>
      <div>
        <a>
          <div
            class="mainContainer-0-2-7"
          >
            <svg
              fill="currentColor"
              height="1em"
              stroke="currentColor"
              stroke-width="0"
              viewBox="0 0 448 512"
              width="1em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M224 256c70.7 0 128-57.3 128-128S294.7 0 224 0 96 57.3 96 128s57.3 128 128 128zm89.6 32h-16.7c-22.2 10.2-46.9 16-72.9 16s-50.6-5.8-72.9-16h-16.7C60.2 288 0 348.2 0 422.4V464c0 26.5 21.5 48 48 48h352c26.5 0 48-21.5 48-48v-41.6c0-74.2-60.2-134.4-134.4-134.4z"
              />
            </svg>
            <div
              class="title-0-2-8 title-d1-0-2-10"
            >
              About
            </div>
          </div>
        </a>
      </div>
      <div>
        <a>
          <div
            class="mainContainer-0-2-7"
          >
            <svg
              fill="currentColor"
              height="1em"
              stroke="currentColor"
              stroke-width="0"
              viewBox="0 0 512 512"
              width="1em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M96.9 270.3V363c0 2.9 1.5 5.5 4 7l132 75.9c5.3 3.1 12-.8 12-7v-93.8c0-2.9-1.5-5.5-4-7l-132-74.9c-5.4-2.9-12 1-12 7.1zM280.9 445.9L413 370c2.5-1.4 4-4.1 4-7v-93.7c0-6.2-6.6-10-12-7l-132 75.9c-2.5 1.4-4 4.1-4 7V439c-.1 6.1 6.6 10 11.9 6.9z"
              />
              <path
                d="M249 65.1L37 188.9c-5.4 3.1-5.4 10.8 0 13.9l212 117.8c4.9 2.8 11 2.8 15.9 0L453 212.9c5.3-3.1 7 .8 7 7v153.4c0 6.8 3.9 10 11 10 4.4 0 10-3.2 10-10V201.5c0-2.9-1.5-5.5-4-7L264.9 65.1c-4.9-2.8-11-2.8-15.9 0z"
              />
            </svg>
            <div
              class="title-0-2-8 title-d2-0-2-11"
            >
              Education
            </div>
          </div>
        </a>
      </div>
      <div>
        <a>
          <div
            class="mainContainer-0-2-7"
          >
            <svg
              fill="currentColor"
              height="1em"
              stroke="currentColor"
              stroke-width="0"
              viewBox="0 0 640 512"
              width="1em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M640 264v-16c0-8.84-7.16-16-16-16H344v-40h72c17.67 0 32-14.33 32-32V32c0-17.67-14.33-32-32-32H224c-17.67 0-32 14.33-32 32v128c0 17.67 14.33 32 32 32h72v40H16c-8.84 0-16 7.16-16 16v16c0 8.84 7.16 16 16 16h104v40H64c-17.67 0-32 14.33-32 32v128c0 17.67 14.33 32 32 32h160c17.67 0 32-14.33 32-32V352c0-17.67-14.33-32-32-32h-56v-40h304v40h-56c-17.67 0-32 14.33-32 32v128c0 17.67 14.33 32 32 32h160c17.67 0 32-14.33 32-32V352c0-17.67-14.33-32-32-32h-56v-40h104c8.84 0 16-7.16 16-16zM256 128V64h128v64H256zm-64 320H96v-64h96v64zm352 0h-96v-64h96v64z"
              />
            </svg>
            <div
              class="title-0-2-8 title-d3-0-2-12"
            >
              Experience
            </div>
          </div>
        </a>
      </div>
      <div>
        <a>
          <div
            class="mainContainer-0-2-7"
          >
            <svg
              fill="currentColor"
              height="1em"
              stroke="currentColor"
              stroke-width="0"
              viewBox="0 0 512 512"
              width="1em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M512 144v288c0 26.5-21.5 48-48 48H48c-26.5 0-48-21.5-48-48V144c0-26.5 21.5-48 48-48h88l12.3-32.9c7-18.7 24.9-31.1 44.9-31.1h125.5c20 0 37.9 12.4 44.9 31.1L376 96h88c26.5 0 48 21.5 48 48zM376 288c0-66.2-53.8-120-120-120s-120 53.8-120 120 53.8 120 120 120 120-53.8 120-120zm-32 0c0 48.5-39.5 88-88 88s-88-39.5-88-88 39.5-88 88-88 88 39.5 88 88z"
              />
            </svg>
            <div
              class="title-0-2-8 title-d4-0-2-13"
            >
              Photography
            </div>
          </div>
        </a>
      </div>
      <div>
        <a>
          <div
            class="mainContainer-0-2-7"
          >
            <svg
              fill="currentColor"
              height="1em"
              stroke="currentColor"
              stroke-width="0"
              viewBox="0 0 512 512"
              width="1em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M431 320.6c0-4.4 1.2-8.6 3.3-12.2.6-1.1 1.4-2.1 2.1-3.1 17.4-26 27.6-57.1 27.6-90.3.3-92.2-77.5-167-173.7-167-83.9 0-153.9 57.1-170.3 132.9-2.4 11.1-3.7 22.4-3.7 34.2 0 92.3 74.8 169.1 171 169.1 15.3 0 35.9-4.6 47.2-7.7 11.3-3.1 22.5-7.2 25.4-8.3 2.9-1.1 6.1-1.7 9.3-1.7 3.6 0 7 .7 10.1 2l56.7 20.1s2.4 1 3.9 1c4.4 0 8-3.5 8-8 0-1-.5-2.7-.5-2.7L431 320.6z"
              />
              <path
                d="M318.5 392.5c-3.6 1-8.2 2.1-13.2 3.2-10.5 2.2-23.9 4.5-34 4.5-96.2 0-171-76.8-171-169.1 0-6.6.7-15 1.5-21.4.6-4.3 1.3-8.6 2.3-12.8 1-4.5 2.2-9 3.5-13.4l-8 7.1C66.8 219.2 48 260 48 302.5c0 29.3 8.5 57.5 24.8 82 2.3 3.5 3.6 6.2 3.2 8-.4 1.8-11.9 62-11.9 62-.6 2.9.5 5.8 2.7 7.7 1.5 1.2 3.3 1.8 5.1 1.8 1 0 2-.2 2.9-.6l56.1-22.1c1.8-.7 3.7-1.1 5.7-1.1 0 0 2.4-.2 6.3 1.3 18.9 7.4 39.8 12 60.7 12 46.6 0 90.4-20.1 120.1-55.1 0 0 3.2-4.4 6.9-9.6-3.7 1.3-7.9 2.6-12.1 3.7z"
              />
            </svg>
            <div
              class="title-0-2-8 title-d5-0-2-14"
            >
              Contact
            </div>
          </div>
        </a>
      </div>
    </div>
  </div>
  <div>
    <div
      name="Home"
    >
      <div
        class="mainContainer-0-2-15"
      >
        <div
          class="childContainer-0-2-17"
        >
          <h1
            class="textColor-0-2-16"
          >
            Zeeshan
          </h1>
          <div
            class="textColor-0-2-19"
          >
            I am a
            <span
              class="animatedText-0-2-20"
            >
              <div
                class="x0 x1 x2 "
              >
                <div
                  style="transition: width 150ms linear; height: auto; width: auto;"
                >
                  <div
                    class="x0 x3 x4 x5"
                    style="opacity: 1; transform: translateY(0px); position: relative;"
                  >
                    Front End Developer
                  </div>
                </div>
              </div>
            </span>
          </div>
          <div
            class="mainContainer-0-2-21"
          >
            <div
              class="socialIconContainer-0-2-22"
            >
              <svg
                fill="currentColor"
                height="25"
                size="25"
                stroke="currentColor"
                stroke-width="0"
                viewBox="0 0 496 512"
                width="25"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"
                />
              </svg>
            </div>
            <div
              class="socialIconContainer-0-2-22"
            >
              <svg
                fill="currentColor"
                height="25"
                size="25"
                stroke="currentColor"
                stroke-width="0"
                viewBox="0 0 448 512"
                width="25"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M416 32H31.9C14.3 32 0 46.5 0 64.3v383.4C0 465.5 14.3 480 31.9 480H416c17.6 0 32-14.5 32-32.3V64.3c0-17.8-14.4-32.3-32-32.3zM135.4 416H69V202.2h66.5V416zm-33.2-243c-21.3 0-38.5-17.3-38.5-38.5S80.9 96 102.2 96c21.2 0 38.5 17.3 38.5 38.5 0 21.3-17.2 38.5-38.5 38.5zm282.1 243h-66.4V312c0-24.8-.5-56.7-34.5-56.7-34.6 0-39.9 27-39.9 54.9V416h-66.4V202.2h63.7v29.2h.9c8.9-16.8 30.6-34.5 62.9-34.5 67.2 0 79.7 44.3 79.7 101.9V416z"
                />
              </svg>
            </div>
            <div
              class="socialIconContainer-0-2-22"
            >
              <svg
                fill="none"
                height="25"
                size="25"
                stroke="currentColor"
                stroke-linecap="round"
                stroke-linejoin="round"
                stroke-width="2"
                viewBox="0 0 24 24"
                width="25"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M23 3a10.9 10.9 0 0 1-3.14 1.53 4.48 4.48 0 0 0-7.86 3v1A10.66 10.66 0 0 1 3 4s-4 9 5 13a11.64 11.64 0 0 1-7 2c9 5 20 0 20-11.5a4.5 4.5 0 0 0-.08-.83A7.72 7.72 0 0 0 23 3z"
                />
              </svg>
            </div>
            <div
              class="socialIconContainer-0-2-22"
            >
              <svg
                fill="currentColor"
                height="25"
                size="25"
                stroke="currentColor"
                stroke-width="0"
                viewBox="0 0 448 512"
                width="25"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M400 32H48A48 48 0 0 0 0 80v352a48 48 0 0 0 48 48h137.25V327.69h-63V256h63v-54.64c0-62.15 37-96.48 93.67-96.48 27.14 0 55.52 4.84 55.52 4.84v61h-31.27c-30.81 0-40.42 19.12-40.42 38.73V256h68.78l-11 71.69h-57.78V480H400a48 48 0 0 0 48-48V80a48 48 0 0 0-48-48z"
                />
              </svg>
            </div>
            <div
              class="socialIconContainer-0-2-22"
            >
              <svg
                fill="none"
                height="25"
                size="25"
                stroke="currentColor"
                stroke-linecap="round"
                stroke-linejoin="round"
                stroke-width="2"
                viewBox="0 0 24 24"
                width="25"
                xmlns="http://www.w3.org/2000/svg"
              >
                <rect
                  height="20"
                  rx="5"
                  ry="5"
                  width="20"
                  x="2"
                  y="2"
                />
                <path
                  d="M16 11.37A4 4 0 1 1 12.63 8 4 4 0 0 1 16 11.37z"
                />
                <line
                  x1="17.5"
                  x2="17.5"
                  y1="6.5"
                  y2="6.5"
                />
              </svg>
            </div>
            <div
              class="socialIconContainer-0-2-22"
            >
              <svg
                fill="currentColor"
                height="25"
                size="25"
                stroke="currentColor"
                stroke-width="0"
                viewBox="0 0 496 512"
                width="25"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M248 8C111 8 0 119 0 256s111 248 248 248 248-111 248-248S385 8 248 8zm121.8 169.9l-40.7 191.8c-3 13.6-11.1 16.9-22.4 10.5l-62-45.7-29.9 28.8c-3.3 3.3-6.1 6.1-12.5 6.1l4.4-63.1 114.9-103.8c5-4.4-1.1-6.9-7.7-2.5l-142 89.4-61.2-19.1c-13.3-4.2-13.6-13.3 2.8-19.7l239.1-92.2c11.1-4 20.8 2.7 17.2 19.5z"
                />
              </svg>
            </div>
            <div
              class="socialIconContainer-0-2-22"
            >
              <svg
                fill="currentColor"
                height="25"
                size="25"
                stroke="currentColor"
                stroke-width="0"
                viewBox="0 0 1024 1024"
                width="25"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M928 160H96c-17.7 0-32 14.3-32 32v640c0 17.7 14.3 32 32 32h832c17.7 0 32-14.3 32-32V192c0-17.7-14.3-32-32-32zm-40 110.8V792H136V270.8l-27.6-21.5 39.3-50.5 42.8 33.3h643.1l42.8-33.3 39.3 50.5-27.7 21.5zM833.6 232L512 482 190.4 232l-42.8-33.3-39.3 50.5 27.6 21.5 341.6 265.6a55.99 55.99 0 0 0 68.7 0L888 270.8l27.6-21.5-39.3-50.5-42.7 33.2z"
                />
              </svg>
            </div>
          </div>
          <div
            class="buttonContainer-0-2-18"
          >
            <button
              class="mainContainer-0-2-23"
              style="transform: perspective(600px) rotateX(0deg) rotateY(0deg) scale(1);"
            >
              Hire me
            </button>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div>
    <div
      name="About"
    >
      <div
        class="header-0-2-24"
      >
        <div
          style="opacity: 0; transform: translate3d(0,20px,0);"
        >
          <h2
            class="h2Text-0-2-30"
          >
            About
          </h2>
        </div>
        <div
          style="opacity: 0; transform: translate3d(0,20px,0);"
        >
          <h2
            class="h2Text-0-2-30"
          >
            Me
          </h2>
        </div>
      </div>
      <div
        class="aboutContainer-0-2-28"
      >
        <div
          class="avatar-0-2-25"
        />
        <div
          class="aboutInfo-0-2-26"
        >
          <div
            class="mainContainer-0-2-31"
          >
            <div
              class="triangleContainer-0-2-40"
            >
              <div
                class="triangle-0-2-41"
              />
            </div>
            <div
              class="descriptionParentContainer-0-2-44"
            >
              <div
                class="descriptionContainer-0-2-32"
              >
                <div
                  class="description-0-2-34"
                >
                  I am Shaikh Zeeshan Murshed, Front-End developer from Montreal, Canada. I have experience in making Website. Also I am a hobbyist photographer.
                </div>
                <div
                  class="button-0-2-36"
                >
                  <button
                    class="mainContainer-0-2-23"
                    style="transform: perspective(600px) rotateX(0deg) rotateY(0deg) scale(1);"
                  >
                    Download My CV
                  </button>
                </div>
              </div>
              <div
                class="skillsInfoContainer-0-2-38"
              />
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div />
  <div />
  <div>
    <div
      name="Photography"
    >
      <div
        class="header-0-2-46"
      >
        <div
          style="opacity: 0; transform: translate3d(0,20px,0);"
        >
          <h2
            class="h2Text-0-2-30"
          >
            Photography
          </h2>
        </div>
      </div>
      <div
        class="photographyContainer-0-2-47"
      >
        <div
          class="mainContainer-0-2-49"
        >
          <div
            class="photosContainer-0-2-51"
          />
          <div
            class="button-0-2-52"
          >
            <button
              class="mainContainer-0-2-23"
              style="transform: perspective(600px) rotateX(0deg) rotateY(0deg) scale(1);"
            >
              Explore more images
            </button>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div>
    <div
      name="Contact"
    >
      <div
        class="header-0-2-53"
      >
        <div
          style="opacity: 0; transform: translate3d(0,20px,0);"
        >
          <h2
            class="h2Text-0-2-30"
          >
            Contact
          </h2>
        </div>
      </div>
      <div
        class="contactUsContainer-0-2-57"
      >
        <div
          class="mapContainer-0-2-55"
        >
          <div
            class="sc-AxjAm faZJjB"
          >
            <div
              style="position: relative; width: 0px; height: 0px; cursor: grab; outline: none;"
              tabindex="0"
            >
              <div
                style="position: relative; width: 100%; height: 100%;"
              >
                <div
                  class=""
                  style="position: absolute; width: 100%; height: 100%; overflow: hidden; visibility: inherit;"
                />
                <div
                  style="overflow: visible; height: 0px; width: 0px;"
                />
                <div
                  class="resize-triggers"
                >
                  <div
                    class="expand-trigger"
                  >
                    <div
                      style="width: 1px; height: 1px;"
                    />
                  </div>
                  <div
                    class="contract-trigger"
                  />
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          class="formContainer-0-2-54"
        >
          <form
            class="form-0-2-59"
          >
            <input
              class="smallInputField-0-2-60"
              name="name"
              placeholder="Your Name"
              type="text"
              value=""
            />
            <input
              class="smallInputField-0-2-60"
              name="email"
              placeholder="Email address"
              type="email"
              value=""
            />
            <input
              class="smallInputField-0-2-60"
              name="subject"
              placeholder="Subject"
              type="text"
              value=""
            />
            <textarea
              class="smallInputField-0-2-60 textArea-0-2-61"
              name="message"
              placeholder="Type your message here."
            />
            <div
              class="buttonContainer-0-2-62"
            >
              <div>
                <button
                  class="mainContainer-0-2-23"
                  style="transform: perspective(600px) rotateX(0deg) rotateY(0deg) scale(1);"
                >
                  Send Message
                </button>
                <div
                  class="emailTextContainer-0-2-63"
                >
                  Don't like forms? Send me an
                  <span
                    class="email-0-2-64"
                  >
                    Email
                  </span>
                </div>
              </div>
            </div>
          </form>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
